indx.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Abbas Aljuaifari</title>
+
+      <!--stylesheet-->
+
+    <link rel="stylesheet" href="style.css">
+
+     <!--google fonts-->
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer">
+
+</head>
+<body>
+
+    <!--Header with navbar starts here-->
+
+    <header class="header" id="header">
+        <a href="#"><img src="images/MyLogo.png"></a>
+
+        <!-- toggle button -->
+        <nav>
+            <button type="button" class="menu-button">
+                <span class="open" id="menu"></span>
+            </button>
+        </nav>
+    </header>
+
+     <!--Header with navbar ends here-->
+
+       <!-- main starts here -->
+
+    <main>
+        <section class="home-section" id="home">
+            <div class="home-pag">
+                <div>
+
+                    <h1 class="home-pag-header">Hey There.</h1>
+                    <h1 class="home-pag-header">I'm Abbas</h1>
```

Comment:
- [ ] I'll suggest you use only one `h1` tag to wrap the two texts which is the main title of your project. Following the best practices, you are only meant to have one `h1` tag on your web page. :smile: :+1: